dynamic_programming/range_maximum_query.py

```diff
@@ -0,0 +1,97 @@
+"""
+Range Maximum Query Problem:
+
+    Given an array of integers and q queries, for each query, find the maximum value in the range [l, r).
+    The array is 0-indexed and the queries are 0-indexed.
+
+    More details:
+        https://www.geeksforgeeks.org/range-maximum-query-using-sparse-table/
+
```

Comment:
```suggestion
```

Reply:
I think "More details: https://www.geeksforgeeks.org/range-maximum-query-using-sparse-table/" is not necessary to be here.